src/Compilers/VisualBasic/Portable/Symbols/NamedTypeSymbol.vb

```diff
@@ -1089,6 +1099,85 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         ''' </summary>
         Friend MustOverride Function GetSynthesizedWithEventsOverrides() As IEnumerable(Of PropertySymbol)
 
+        Friend Overridable ReadOnly Property AllRequiredMembers As ImmutableSegmentedDictionary(Of String, Symbol)
```

Comment:
>Overridable [](http://example.com/codeflow?start=15&length=11)

It looks like the default implementation doesn't use ```AllRequiredMembers``` and ```HasRequiredMembersError``` in the process of calculation, and, therefore, going to ignore the overridden behavior while traversing base types. I am going to mention once again that I think that we should not use recursion through ```TryCalculateRequiredMembersWithCache``` while traversing base types. We should use these properties instead.

Reply:
I'm resolving this by simply removing the overrides.